spec/javascripts/components/story/story_description_spec.js

```diff
@@ -29,7 +30,7 @@ describe('<StoryDescription />', function() {
         value={""}
       />
     );
-    expect(wrapper.find('.description')).toHaveText('Description');
+    expect(wrapper.find('.description').text()).toContain('Description');
```

Comment:
You should add an expectation of `makeHtml` being called with the right arguments. Like `expect(window.md.makeHtml).to.have.been.calledWith(your agument here)` to make it less brittle. And use the same mindset for the other tests where you force the return of `makeHtml` 😄

Reply:
Alright ;D